dbt/include/snowflake/macros/adapters.sql

```diff
@@ -155,14 +155,41 @@
     show terse objects in {{ schema_relation }}
   {%- endset -%}
 
+  {%- set fallback_sql -%}
+     select
+      created as "created_on",
+      table_name as "name",
+      case when table_type = 'BASE TABLE' then 'table'
```

Comment:
Minor nit, and not sure if this works in snowflake sql, but these type of case statements can also usually be written this way:
```sql
case table_type
    when 'BASE TABLE' then 'table'
    when 'VIEW' then 'view'
    when 'MATERIALIZED VIEW' then 'materializedview'
    when 'EXTERNAL TABLE' then 'external'
    else table_type
end as "kind"
```

Reply:
Hey @mikealfare thanks for reviewing this. I learned late yesterday that in fact we can paginate the `show terse objects` query on snowflake, so shifted direction there in working with @dbeatty10.